onmt/Translator.py

```diff
@@ -6,6 +6,28 @@
 from torch.autograd import Variable
 
 
+class DecoderState(object):
```

Comment:
I think this should be in model and not here, unfortunately. We need to think a bit more about the names of repeatBeam and beamUpdate.